python/lsst/ip/isr/ptcDataset.py

```diff
@@ -811,6 +817,15 @@ def fromTable(cls, tableList):
                 inDict['ampOffsets'][ampName] = record['AMP_OFFSETS']
                 inDict['gainUnadjusted'][ampName] = record['GAIN_UNADJUSTED']
                 inDict['gainList'][ampName] = record['GAIN_LIST']
+            if calibVersion < 2.1:
+                inDict['covariancesModelNoB'][ampName] = record['COVARIANCES_MODEL']
+                inDict['aMatrixNoB'][ampName] = record['A_MATRIX']
+            else:
+                inDict['covariancesModelNoB'][ampName] = np.full_like(
+                    inDict['covariancesModel'][ampName],
+                    np.nan,
+                )
+                inDict['aMatrixNoB'][ampName] = np.full_like(inDict['aMatrix'][ampName], np.nan)
```

Comment:
I don't know if this else is necessary.  For newer PTCs we shouldn't be using these fields at all, right?